requirements.txt

```diff
@@ -9,3 +9,4 @@ pytest-cov
 mock
 coverage
 flake8>=3.8.4
+ase
```

Comment:
Do we really want to put ase as a requirement?
I don't think so.
But if we do it, I would specify a version.

Reply:
Do you have a specific case in mind where specifying the format is important? In my experience, it's mostly irrelevant - ASE internally checks the file name and deduces it from the extension, which in virtually all cases will be enough.

Reply:
(sorry, this comment of course referred to your first observation about the use of the method `read`, not the version/dependency thing)

Reply:
I believe it was me adding ASE to the requirements - problem is that if we don't the CI fails. if you can make it a requirement only for the CI, then I'm happy to drop it here, but the test will fail when launched locally if ASE is not there

Reply:
Can't we make the test itself optional? Just skips it altogether if an attempt to load ASE gives an `ImportError`.

Reply:
that would be ideal but I think we don't have the infrastructure for that. @litman90 might be in a better position to tweak the tests infrastructure in this direction.